Userland/Libraries/LibWeb/HTML/HTMLDialogElement.idl

```diff
@@ -1,11 +1,15 @@
 #import <HTML/HTMLElement.idl>
 
 // https://html.spec.whatwg.org/multipage/semantics.html#htmldialogelement
-[Exposed=Window]
+[Exposed=Window, UseNewAKString]
```

Comment:
This annotation can be removed now, AK::String is now the default.

Reply:
Oh great! :catdog: